### ISSUE TITLE
[TESTING] Quest Four: Staff of the Avatar

### ISSUE THREAD

**Issue by cambragol** [cambragol/advent-of-the-trinity#216]
This quest needs to be tested for quality and completeness.

This is a quest to get the staff found on the cover of the box. It is a long quest, than can only be solved by following various small crumbs of clues. There are also several ways to solve it. It is long and potentially tough, but with a huge reward. This one gets started after talking with Lord British's now more conversational court bard.

**Comment by MrKnitster** [comment]
Off to a fine start.

<details> 
  <summary>Spoiler</summary>
  The bard gives the background, and LB answers to STAFF to suggest its power. But not sure where to go from here. I'll just start asking around. Maybe a tavern clue? Maybe a sailor? I have the names of the ship and maker of the staff, but no leads on where to go next.
</details>

**Comment by cambragol** [comment]
This one could be hard. I would say potentially it is the hardest in TotT and even default QotA.

General knowledge of Britannia, and its towns may come in handy here too.

**Comment by MrKnitster** [comment]
Hmmm. Might be stumped on this one. Found the trail but then went cold and don't know where to head next.

<details> 
  <summary>Spoiler</summary>
  Visited Wreck of the Hyaline and talked to everyone (I think), but no one had any further clues. I must be missing something. 
</details>

**Comment by cambragol** [comment]
This is a hard quest. The hardest. It does not follow a simple 'talk to X then go to Y and talk to Z' format. You will have to gather clues, then deduce what happened. You will also need knowledge of new things in the game, particularly things revolving around the 'lockpick kit' quest, and the Rogue King.

**Comment by cambragol** [comment]
There is a weak point in this quest line that obviously stumped you. Have you tried speaking to the guard in the 
armory at castle Britannia?

**Comment by MrKnitster** [comment]
Well, there's a guy I never thought of. I'll give it a try. Does someone lead to him? I did find some clues generally on the item's path.

**Comment by cambragol** [comment]
No, no one leads to him. I just thought it would be a logical place to pick-up/start the quest trail. He was expecting the staff to be placed in the armory, until the Avatar came to claim it.

**Comment by MrKnitster** [comment]
Yes, this was hard, but in the end, I think well balanced. It is solvable, even with the difficulty, and even if players don't actually talk to every person involved. By not having it a chain of a-to-b-to-c etc., you eliminate the risk of losing the thread. This quest has to be approached much more generally. Very fun in the end, and satisfying to crack.

Now, what does it do? Off to experiment and find out!

**Comment by cambragol** [comment]
Good. I wanted it to be hard enough that your average quester might fail to solve it. But a persistent approach would yield success. Like you say, you don't need to know every piece of the chain...just a little logical/deductive reasoning, and general world experience/knowledge, can open up the quest.

As for the function of the Staff....oops...I don't think I made that very clear did I...that might need to be fixed. As it was a commissioned gift for the avatar, someone should know its power...(It is VERY powerful). 
Remember, I added this item to help with the difficulty of the underdeep quest.

**Comment by MrKnitster** [comment]
I'll have to play around with this. So far, I have not been able to see any effect. [U]sing it does nothing, and I haven't seen other things change (spell cost, duration, other enhancements). I'll let you know if I notice anything.

**Comment by cambragol** [comment]
I really should have a simple explanation somewhere. I didn't intend for it to be a mystery to be solved. Ideally it would be something that Lord British tells you. However, I can't fit any dialogue in there. So I might have to fit it in with someone else...

The Staff augments your power, allowing ANY spell effect to be extended 5x longer. So use it, then cast another spell, and the effect should be extended.

**Comment by MrKnitster** [comment]
Hmm. When you use the staff, you get the usual "Hmmm No Effect" message. That seems a little misleading. Maybe there should be a message that implies a power boost? Ripple in the ether?

**Comment by cambragol** [comment]
You used it and you got that message? You should see this:

![AVATAR EXE 2021-11-19 at 6 44 50 AM](https://user-images.githubusercontent.com/5019739/142502449-db7ee131-93b2-409a-8193-ca27e7985f03.png)

That message (No Effect) usually comes from yelling an ineffective word. For items that are unusable you should get 'Not Usable Item!"

Something is odd here...

**Comment by MrKnitster** [comment]
Let me try again and confirm. Thanks!

**Comment by cambragol** [comment]
Ah! You are not an Avatar!!

**Comment by MrKnitster** [comment]
Oh, of course! That makes perfect sense.

**Comment by cambragol** [comment]
whew!

**Comment by cambragol** [comment]
Actually, Lord British does lay out its power. He says 'It was to be used with thy Ankh, to augment thy magics'

Is this a little vague though? I suppose it must be a little.

**Comment by cambragol** [comment]
I tweaked the bard Juel's dialogue, to give a little more specific description of the Staff's power. This should help.

**Comment by MrKnitster** [comment]
Excellent. Now that you remind me of LB's comment, I think this is good. I've been holding off on avatarhood in order to keep the whirlpool spawn rate strong (haven't sealed any dungeons yet either), but I'm ready to gain a few eighths.

**Comment by cambragol** [comment]
Hmm...not gaining avatarhood to keep whirlpool spawns strong is not the kindof comment I want to be seeing. Perhaps I should release another version with the spawn rate I have tweaked. I was hoping the next release to be a 'release candidate'.
